client/src/components/MagicalFactor/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import Keywords from '../../assets/Keywords';
+
+import './style.css';
+
+const MagicalFactor = ({ value: { text, src }, reviewText }) => (
+  <div className="magical">
+    <div className="magical__keyword">
+      <div className="magical__keyword__img">
+        <img src={Keywords[src]} alt={text} className="magical__keyword__img" />
+      </div>
+      <div
+        className={`magical__keyword__div${text.length > 18 ? ' smaller' : ''}`}
```

Comment:
`magical__keyword-div` or better `magical__keyword-text` or `magical__keyword-label`

Reply:
and, `smaller` is a modifier 🙂 so `magical__keyword-text${text.length > 18 ? ' --smaller' : ''}`

Reply:
yes ik, but it felt a bit redundant,
fixed